feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts

```diff
@@ -73,33 +116,27 @@ export class ImportEntriesDialogComponent {
     return form;
   }
 
-  selectFile(file: File, form: FormGroup) {
-    this.selectedFile = file;
-    this.importService.loadFile(file).subscribe(
-      (data) => {
-        if (this.importToCartService.isDataParsable(data)) {
-          this.fileError = {};
-          this.loadedFile = data as string[][];
-        } else {
-          this.fileError = { notParsable: true };
-          this.loadedFile = null;
-        }
-        form.get('file')?.updateValueAndValidity();
-      },
-      (error) => {
-        this.fileError = error;
-        this.loadedFile = null;
-        form.get('file')?.updateValueAndValidity();
-      }
-    );
+  protected validateMaxSize(file: File) {
+    if (
+      this.fileValidity?.maxSize &&
+      file.size / 1000000 > this.fileValidity?.maxSize
```

Comment:
Why we dividing size by `1000000`? Can we put a comment here? Or maybe assign such value to some const which will describe what it means.

Reply:
There is already comment in interface